### PR TITLE
Fix and improve layout for Projects page

### PR DESCRIPTION
**Pull request by frozenhelium**
* Move sidebar to a different component
* Improve padding in sidebar to make the view more comfy
* Move the Form back to General tab from Detail component
* Allocate space for visualization in general tab
* Add a separate font for headings
* Set default color of headings to primary color
* Change value of primary color to slightly darker shade

### CODE REVIEW

File: src/views/Project/Details/Frameworks/EditFramework/index.js [hunk 1]
```diff
@@ -137,20 +138,6 @@ export default class ProjectAfDetail extends React.PureComponent {
                         placeholder={_ts('project', 'projectDescriptionPlaceholder')}
                         rows={3}
                     />
-                    <div className={styles.actionButtons}>
```
Comment (AdityaKhatri) [inline src/views/Project/Details/Frameworks/EditFramework/index.js]: Include buttons inside faram

File: src/views/Project/Details/General/index.js [hunk 2]
```diff
@@ -1,65 +1,269 @@
...
+            pendingProjectPut: false,
+        };
+
+        this.schema = {
+            fields: {
+                title: [requiredCondition],
+                startDate: [dateCondition],
+                endDate: [dateCondition],
+                description: [],
+                regions: [],
```
Comment (AdityaKhatri) [inline src/views/Project/Details/General/index.js]: regions, usergroups and memberships are not in the form

File: src/views/Project/Details/General/index.js [hunk 3]
```diff
@@ -1,65 +1,269 @@
...
+            message={
+                (location) => {
+                    const {
+                        routeUrl,
+                        projectLocalData: {
+                            pristine,
+                        },
+                    } = this.props;
+
+                    console.warn(pristine, location.pathname === routeUrl);
```
Comment (AdityaKhatri) [inline src/views/Project/Details/General/index.js]: remove console.warn
Also, prompt during hash change

File: src/views/Project/Details/General/index.js [hunk 4]
```diff
@@ -1,65 +1,269 @@
...
-                    label={_ts('project.general', 'projectDescriptionLabel')}
-                    faramElementName="description"
-                    placeholder={_ts('project.general', 'projectDescriptionPlaceholder')}
-                    className={styles.description}
-                    rows={3}
-                />
-            </div>
+                    { loading && <LoadingAnimation /> }
+                    <div className={styles.visualizations}>
+                        Visualizations
```
Comment (AdityaKhatri) [inline src/views/Project/Details/General/index.js:204]: Use strings or write fixme

Comment (frozenhelium) [reply src/views/Project/Details/General/index.js:204]: Just a placeholder, Not meant to be there (Robot detected)

Comment (AdityaKhatri) [reply src/views/Project/Details/General/index.js:204]: If just a placeholder you should write FIXME / TODO.
*beep boop beep boop*

**Pull request merged**